pulp_deb/app/tasks/publishing.py

```diff
@@ -413,3 +431,10 @@ def _zip_file(file_path):
         with GzipFile(gz_file_path, "wb") as f_out:
             shutil.copyfileobj(f_in, f_out)
     return gz_file_path
+
+
+def _fetch_file_checksum(file_path, index, allowed_checksum):
+    h = getattr(index.contentartifact_set.first().artifact, allowed_checksum)
+    checksum_type = CHECKSUM_TYPE_MAP[allowed_checksum]
```

Comment:
Nitpick: This function really does not care if the checksum being passed in is an allowed checksum, so my naming preference would be:
```suggestion
def _fetch_file_checksum(file_path, index, checksum):
    h = getattr(index.contentartifact_set.first().artifact, checksum)
    checksum_type = CHECKSUM_TYPE_MAP[checksum]
```

Reply:
Done.